api/pom.xml

```diff
@@ -58,15 +58,11 @@
 			<type>jar</type>
 			<scope>test</scope>
 		</dependency>
-		
-		<dependency>
-			<groupId>org.mockito</groupId>
-			<artifactId>mockito-all</artifactId>
-		</dependency>
 
 		<dependency>
-			<groupId>org.codehaus.groovy</groupId>
-			<artifactId>groovy-all</artifactId>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>reportingcompatibility-api</artifactId>
+			<version>2.0.6</version>
```

Comment:
Shouldn't we add scope?

Reply:
I added the scope

Reply:
Looking at this again, do you mean the calculation module depends on the reportingcompatibility module?

Reply:
PatientSetService was removed in OpenMRS Platform 2.0.0, so I used the reporting compatibility module to replace it.

Replaced:

`Context.getPatientSetService().getObservations(new Cohort(cohort), whichConcept); `

with

`Context.getService(ReportingCompatibilityService.class).getObservations(new Cohort(cohort), whichConcept);`

https://openmrs.atlassian.net/browse/TRUNK-4874
https://talk.openmrs.org/t/we-should-remove-patientsetservice-in-openmrs-core-in-platform-2-0/6173

Reply:
The calculation module is supposed to be very light weight and hence should not depend on the reporting compatibility module. The reportingcompatibilty module is being deprecated.

Reply:
@dkayiwa I removed the reporting compatibility module from this